icons/solidjs/wordmark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="306.42" height="70.7"><defs><clipPath id="a"><path style="fill:none" d="M0 0h306.42v70.7H0z"/></clipPath></defs><g style="clip-path:url(#a)"><path d="M33.9 45a12 12 0 0 0 8.1 3.4c3.4 0 6.2-2 6.2-5.4 0-7.8-15.5-5.9-15.5-15.5 0-4.5 4-8.3 9.7-8.3a12 12 0 0 1 8.3 3l-1.5 2.6a10.38 10.38 0 0 0-6.8-2.7c-4 0-6.5 2.6-6.6 5.3 0 7.4 15.5 5.3 15.5 15.5 0 4.6-3.6 8.4-9.4 8.4a13.36 13.36 0 0 1-9.8-4ZM83 19.4a15.73 15.73 0 0 1 15.8 15.9A16 16 0 0 1 83 51.5h-.1a16 16 0 0 1-15.8-16.2v-.1A15.73 15.73 0 0 1 83 19.4m-.1 29.3a13 13 0 0 0 12.7-13.29v-.11c0-7.4-5.6-13.1-12.6-13.1a12.72 12.72 0 0 0-12.7 13c0 7.6 5.6 13.4 12.6 13.5M116.8 20h3l-.1 28.3h14.4V51h-17.5Zm33.9.1h3l-.1 31.1h-3Zm23.2 0h10c9.3 0 15.6 5.7 15.6 15.6s-6.3 15.5-15.6 15.5h-10Zm9.6 28.4c7.6 0 12.7-4.5 12.8-12.8s-5.1-12.8-12.7-12.8h-6.7l-.1 25.6Zm42.17 2.7a10.27 10.27 0 0 0 7.06-2.52 9 9 0 0 0 2.86-7.1V20.47h-6.25v20.65a4.2 4.2 0 0 1-1 3 3.61 3.61 0 0 1-2.77 1.07q-2.94 0-4.29-3.45L216 44.69q2.6 6.51 9.69 6.51m37.44.07c3.47 0 6.19-.86 8.14-2.59a8.63 8.63 0 0 0 2.93-6.79 7.09 7.09 0 0 0-2.5-5.67c-1.68-1.44-4.37-2.68-8.07-3.74a11.76 11.76 0 0 1-3.76-1.57 2.57 2.57 0 0 1-1.1-2.15 2.7 2.7 0 0 1 1.1-2.25 4.6 4.6 0 0 1 2.91-.85 5.67 5.67 0 0 1 5.71 3.72l5.17-2.83a10.12 10.12 0 0 0-4.07-4.85 12.25 12.25 0 0 0-6.54-1.7 11.11 11.11 0 0 0-7.45 2.6 8.11 8.11 0 0 0-3.09 6.55 7.5 7.5 0 0 0 2.32 5.61q2.09 2.1 7.37 3.69a16.51 16.51 0 0 1 4.52 1.82 2.33 2.33 0 0 1 1.19 2.09 2.74 2.74 0 0 1-1.29 2.31 5.64 5.64 0 0 1-3.3.91 7 7 0 0 1-6.6-4.23l-5.4 2.79a10.85 10.85 0 0 0 4.49 5.23 13.74 13.74 0 0 0 7.32 1.9" style="fill:#fff"/></g></svg>
```

Comment:
You don't need that SVG, it doesn't belong to any possible version stated here : https://github.com/devicons/devicon/wiki/SVG-Versions -> Unless you want to make it the `solidjs-line-wordmark` ?

Reply:
My bad, I was adding that SVG to the logo :')

Reply:
This icon needs to be removed from the PR